src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingcar;
+
+public class Car {
+
+    final int INIT_POSITION = 0;
+    String name;
+    int position;
+
+    Car(String name) {
+        this.name = name;
+        this.position = INIT_POSITION;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return this.position;
+    }
+
+    public void go() {
+        this.position++;
+    }
```

Comment:
```suggestion
    public void go() {
        this.position++;
    }
    
    public String getName() {
        return this.name;
    }

    public int getPosition() {
        return this.position;
    }
```

사용하지 않는 메서드는 제거해주세요! 🙏
또한 getter와 setter, toString과 같은 메서드는 클래스의 최 하단에 두는 것이 가독성에 좋아요! 😃